docs/usage/scrapy-poet.rst

```diff
@@ -93,10 +100,47 @@ country code:
 
     from scrapy_zyte_api import Geolocation
 
+
     @attrs.define
     class MyPageObject(BasePage):
         product: Product
         geolocation: Annotated[Geolocation, "DE"]
 
-.. note:: As :class:`~.Geolocation` is only useful when annotated, using it
-    requires Python 3.9+.
+.. _browser-actions:
+
+Browser actions
+---------------
+
+You can specify browser actions by adding a :class:`scrapy_zyte_api.Actions`
+dependency and annotating it with actions passed to the
+:func:`scrapy_zyte_api.actions_list` function:
+
+.. code-block:: python
+
+    from typing import Annotated
+
+    from scrapy_zyte_api import Actions, actions_list
+
+
+    @attrs.define
+    class MyPageObject(BasePage):
+        product: Product
+        actions: Annotated[
+            Actions,
+            actions_list(
+                [
+                    {
+                        "action": "click",
+                        "selector": {"type": "css", "value": "button#openDescription"},
+                        "delay": 0,
+                        "button": "left",
+                        "onError": "return",
+                    },
+                    {"action": "waitForTimeout", "timeout": 5, "onError": "return"},
+                ]
+            ),
+        ]
+
+You can access the results of these actions in the
+:attr:`.Actions.results` attribute of the dependency in the
+resulting page object.
```

Comment:
Could you please add a code example? it'd be nice to show e.g. how to check if action was a success or not.

Reply:
Maybe even in the validate_input method